flume-ng-sinks/flume-ng-elasticsearch-sink/src/test/java/org/apache/flume/sink/elasticsearch/TestElasticSearchIndexRequestBuilderFactory.java

```diff
@@ -24,46 +24,63 @@
 import org.apache.flume.conf.ComponentConfiguration;
 import org.apache.flume.conf.sink.SinkConfiguration;
 import org.apache.flume.event.SimpleEvent;
+import org.apache.flume.sink.elasticsearch.legacy.FastByteArrayOutputStream;
+import org.elasticsearch.action.index.IndexAction;
 import org.elasticsearch.action.index.IndexRequestBuilder;
 import org.elasticsearch.client.Client;
 import org.elasticsearch.common.io.BytesStream;
-import org.elasticsearch.common.io.FastByteArrayOutputStream;
+import org.elasticsearch.common.settings.Settings;
+import org.elasticsearch.common.transport.InetSocketTransportAddress;
+import org.elasticsearch.transport.client.PreBuiltTransportClient;
 import org.junit.Before;
 import org.junit.Test;
 
 import java.io.IOException;
+import java.net.InetAddress;
+import java.net.UnknownHostException;
 import java.util.Map;
 
-import static org.junit.Assert.assertArrayEquals;
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertNull;
-import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.*;
 
 public class TestElasticSearchIndexRequestBuilderFactory
     extends AbstractElasticSearchSinkTest {
 
-  private static final Client FAKE_CLIENT = null;
-
+  private static Client FAKE_CLIENT = null;
   private EventSerializerIndexRequestBuilderFactory factory;
 
   private FakeEventSerializer serializer;
 
   @Before
   public void setupFactory() throws Exception {
+
     serializer = new FakeEventSerializer();
     factory = new EventSerializerIndexRequestBuilderFactory(serializer) {
       @Override
       IndexRequestBuilder prepareIndex(Client client) {
-        return new IndexRequestBuilder(FAKE_CLIENT);
+        Settings settings = Settings.builder()
+            .put("path.data", "target/es-test")
+            .put("path.home", "D:\\dev\\elasticsearch\\elasticsearch-5.2.1")
+            .build();
+
+        try {
+          FAKE_CLIENT = new PreBuiltTransportClient(settings)
+              .addTransportAddress(
+                  new InetSocketTransportAddress(InetAddress.getByName("localhost"), 9300));
+          ;
+        } catch (UnknownHostException e) {
+          e.printStackTrace();
+        }
+
+        return new IndexRequestBuilder(FAKE_CLIENT, IndexAction.INSTANCE);
       }
     };
   }
 
   @Test
   public void shouldUseUtcAsBasisForDateFormat() {
-    assertEquals("Coordinated Universal Time",
-        factory.fastDateFormat.getTimeZone().getDisplayName());
+    String timeZone = factory.fastDateFormat.getTimeZone().getDisplayName();
+    assertTrue("Temps universel coordonné".equals(timeZone) ||
```

Comment:
I think here you should use
```java
assertEquals("Coordinated Universal Time",
        factory.fastDateFormat.getTimeZone().getDisplayName(Locale.ENGLISH));
```

And additionly, I'm not sure whether you see my email.